DEVELOP.md

```diff
@@ -3,11 +3,14 @@
 If you've made it this far, **thank you**! We appreciate your contribution, and hope that this document helps you along the way. If you have any questions or problems, don't hesitate to [file an issue](https://github.com/primer/css/issues/new).
 
 ## Structure
-Primer CSS is published to [npm] as [@primer/css]. Each of Primer CSS's "modules" lives in a subfolder under `src/` with an `index.scss` in it. Generally speaking, the styles are divided into three primary themes:
+Primer CSS is published to [npm] as [@primer/css]. Each of Primer CSS's components lives in a subfolder under `src/` with an `index.scss` in it. Generally speaking, the styles are divided into three primary themes:
+
+* **Base** (`base/`) includes CSS reset, global styles, and typography
+* **Themes** (`color-modes/`) includes color-mode themes from primer-primitives
+* **Core** (`core/`) index file for all components, along with global CSS and utility imports
+* **Support** (`support/`) SCSS variables and mixins for writing Primer CSS (example: spacer vars, breakpoints, font-size and weight)
+* **Utilities** (`utilities/`) SCSS variables, mixins and utility classes (example: `margin` and `padding`)
 
-* **Core** styles (in `core/`) are common dependencies, which include support variables, native element and typography styles, buttons, navigation, tooltips, etc.
-* **Product** styles (in `product/`) are specific to github.com, and include components such as avatars, labels, markdown styles, popovers, and progress indicators.
-* **Marketing** styles (in `marketing/`) are specific to GitHub marketing efforts, including international and event-focused sites as well as the more design-heavy feature pages on github.com. Marketing styles include new colors and button styles, and extend the core typography and whitespace scales.
```

Comment:
If we keep `core/`, should we also keep mentioning `product/` + `marketing/`? I guess initially the idea was that instead of having to import all components at once with

```
@import "@primer/css/index.scss";
```

You could for example exclude `product/` by just import `core/` and `marketing/`:

```
@import "@primer/css/color-modes/index.scss";
@import "@primer/css/core/index.scss";
@import "@primer/css/marketing/index.scss";
```

I was also thinking of removing the `core/`, `product/` folders since they just contain an index and can be confusing. But add additional "bundles" the root like:

```
@primer/css/index.scss // All of Primer CSS
@primer/css/product.scss // meant for product pages
@primer/css/marketing.scss // additional styles for marketing pages
```

Although that would be a braking change and needs some extra communication.

Reply:
You know what? I read the description of these and assumed they were legacy. And then somehow didn't notice they still existed as folders 🤷 I think I wish components were in a `/components` directory, but I don't think its worth changing directory structure at this point. I'll edit the doc!

Reply:
Yes, it bugs me too that components are mixed with other folders. Was also think to prefix non-components with `_`. I think some Ruby convention. So that they would all move to the top and it's easier to separate them.

```
_base
_color-modes
_core
_marketing
_product
_support
_utilities
actionlist
alerts
autocomplete
etc.
```

Maybe when we make some other bigger re-structuring.